recipes/movingpandas/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "v0.1.dev2" %}
+
+package:
+  name: movingpandas
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/m/movingpandas/movingpandas-{{ version }}.tar.gz
+  sha256: 5623a30942242336121a2394e4279861f7c7cf5643aca26fac5bccda49db6461
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - matplotlib
+    - seaborn
+    - pandas
+    - geopandas
+    - rasterio
+    - contextily
+    - descartes
```

Comment:
@ocefpaf are there any good practices regarding which dependencies to list, in case of dependencies that are already dependencies of other dependencies? 

Eg numpy is already a dependency of pandas (and eg in geopandas we don't list that explicitly), but also pandas already of geopandas. So just listing geopandas for those would give the same result, but is that also preferred?

Reply:
and same for rasterio -> contextily

Reply:
It should depend on what the package explicitly uses so if `movingpandas` contains `import numpy` then `numpy` is a direct dependency of this package. Another way to think of it is if `pandas` was to stop depending on `numpy` but still have exactly the same API would users still need to have `numpy` installed to use this package?

Reply:
Thanks for the response! 
In that light, removed rasterio (only dependency of contextily) and also seaborn, but added shapely

Reply:
Thank you! I'll also apply these rules to the setup.py in the PyPI package.

Reply:
Following these rules, descartes can be removed as well. It's a dependency of GeoPandas for plotting polygons.